src/Symfony/Component/Notifier/Bridge/Mastodon/MastodonTransport.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Mastodon;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Quentin Dequippe <quentin@dequippe.tech>
+ *
+ * @see https://docs.joinmastodon.org
+ */
+final class MastodonTransport extends AbstractTransport
+{
+    public function __construct(
+        #[\SensitiveParameter] private readonly string $accessToken,
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null
```

Comment:
one line please

Reply:
Done.

Just for curiosity, is there any automatic code style rule (PHP CS Fixer) on the project for this?